src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -816,6 +831,7 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
           HAS_GRID_TEMPLATES.includes(templateID) &&
           renderShowGridToggle()}
         {renderShowBadgesToggle()}
+        {renderShowZendeskToggle()}
```

Comment:
As above, this should just be rendering the extracted component rather than calling a function here.

Reply:
Backlogged in https://github.com/sparkletown/internal-sparkle-issues/issues/101